corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -301,6 +301,15 @@ def test_form_filter(self):
         form.form_filter = "./edd = '123'"
         self.assertXmlEqual(self.get_xml('form-filter'), app.create_suite())
 
+    def test_module_filter(self):
+        """
+        Ensure module filter gets added correctly
+        """
+        app = Application.wrap(self.get_json('suite-advanced'))
+        module = app.get_module(1)
+        module.module_filter = "/mod/filter = '123'"
+        self.assertXmlEqual(self.get_xml('module-filter'), app.create_suite())
```

Comment:
You can use `self.assertXmlPartialEqual` here to avoid having to put the entire suite.xml file in.

e.g. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/test_suite.py#L446-446